plugins/module_utils/prism/vms.py

```diff
@@ -355,14 +355,17 @@ def _build_spec_boot_config(self, payload, param):
         return payload, None
 
     def _build_spec_gc(self, payload, param):
-        fpath = param["script_path"]
+        if param["script_path"]:
```

Comment:
what if script path is not given?
```suggestion
        if param.get("script_path"):
```